app/allocation/views/view.njk

```diff
@@ -119,7 +119,18 @@
   {% endif %}
 
   {% for slot in allocationPeople.filledSlots %}
+  <section class="app-!-position-relative">
     {{ appCard(slot) }}
+    {% if canAccess('allocation:person:assign') %}
+      <p class="app-!-position-top-right govuk-!-margin-top-3">
```

Comment:
My concern with this would be if a person's name stretches to the edge of the card, does that mean this button will be positioned over their name and over the link to the move?